react/features/base/lib-jitsi-meet/native/RTCPeerConnection.js

```diff
@@ -168,8 +151,8 @@ function _setRemoteDescription(sessionDescription) {
                 }
 
                 this._invokeQueuedOnaddstream(q);
-            },
-            (...args) => {
+            })
+            .catch((...args) => {
```

Comment:
like in the other PR: `then().catch()` chain is not the same thing as `then(success, failure)`